docs/user/ppl/cmd/trendline.rst

```diff
@@ -0,0 +1,63 @@
+=============
+rename
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| Use the ``trendline`` command to calculate the moving average on one or more fields in a search result.
+
+
+Syntax
+============
+trendline <average-type>(<number-of-samples>, <source-field>) AS <target-field> [" " <average-type>(<number-of-samples>, <source-field>) AS <target-field> ]...
```

Comment:
Create a code block for syntax please

Reply:
All other commands are not using a code block, so this is consistent.